stable/postgresql/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: postgresql
-version: 0.15.1
+version: 0.15.2
```

Comment:
new feature added - should it rather be 0.16.0?

Reply:
Well, I hesitated since I don't change the default behavior but that's a new feature so yes, you're right, bumping to 0.16.0.